api/service/networkinfo/service.go

```diff
@@ -118,7 +117,7 @@ func (s *Service) DoService() {
 		select {
 		case peer := <-s.peerInfo:
 			if peer.ID != s.Host.GetP2PHost().ID() && len(peer.ID) > 0 {
-				//				utils.GetLogInstance().Info("Found Peer", "peer", peer.ID, "addr", peer.Addrs, "my ID", s.Host.GetP2PHost().ID())
+				//	utils.GetLogInstance().Info("Found Peer", "peer", peer.ID, "addr", peer.Addrs, "my ID", s.Host.GetP2PHost().ID())
```

Comment:
same format change as my last commit

Reply:
yes, I changed it as it is the file I touched and I did the clean up btw.  In your case, you don't have any other changes in this file, so it's unnecessary for you to touch the file just for cleanup.